app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -102,7 +102,7 @@ class ThreadViewModel @Inject constructor(
             return@liveData
         }
 
-        context.trackUserInfo("nbMessagesInThread", thread.messages.count())
+        sendRelevantMatomoAndSentry(thread)
```

Comment:
Maybe rename to say it sends matomo and sentry about thread's stats or something like that